debian/base/Dockerfile

```diff
@@ -20,8 +20,11 @@ LABEL io.confluent.docker.git.id=$COMMIT_ID
 ARG BUILD_NUMBER=-1
 LABEL io.confluent.docker.build.number=$BUILD_NUMBER
 
-ARG CONFLUENT_DEB_REPO="http://packages.confluent.io"
-ARG APT_ALLOW_UNAUTHENTICATED=false
+# ARG CONFLUENT_DEB_REPO="http://packages.confluent.io"
+ARG CONFLUENT_DEB_REPO="https://s3-us-west-2.amazonaws.com/jenkins-confluent-packages/packaging-3.3.x/35"
```

Comment:
@ewencp These are build args which do not require the default values being set here so we can just remove them and then there's nothing to update for each release